packages/plugin-atoms/src/components/Sizer.tsx

```diff
@@ -0,0 +1,34 @@
+import { useComponentUtils } from '@ima/react-page-renderer';
+import { ComponentPropsWithoutRef, memo } from 'react';
+
+export const Sizer = memo(function SizerComponent({
+  className,
+  height,
+  width,
+  placeholder = false,
+  ...rest
+}: {
+  className?: string;
+  height: number;
+  width: number;
+  placeholder?: boolean;
+} & Omit<ComponentPropsWithoutRef<'div'>, 'placeholder'>) {
+  const { $CssClasses } = useComponentUtils();
+  const sizerClassName = $CssClasses(
```

Comment:
zajímavé, já bych to asi spíš inlinoval ale může být :)

Reply:
fixed